metadeploy/consumers.py

```diff
@@ -94,10 +94,8 @@ def is_known_model(self, model):
         return model in KNOWN_MODELS
 
     def has_good_permissions(self, content):
-        if self.handle_org_special_case(content):
-            return True
-        if self.handle_scratch_org_special_case(content):
-            return True
+        if content["model"] == "org":
+            return self.handle_org_special_case(content)
```

Comment:
@jgerigmeyer This change breaks the `scratchorg` subscription, which depends on the UUID pseudo-auth logic. Should users subscribe to `org` instead, or should this branch also trigger on `scratchorg` subscriptions?

Reply:
`scratchorg` is an actual model, so this check can be handled by the model itself (no need for a special case in the consumer): https://github.com/oddbird/MetaDeploy/pull/145/files#diff-ec7e2c19ed48c380e943554859c3b966R914-R916